### PR TITLE
[prometheus-pushgateway] PriorityClassName for prom-pushgateway

### LINKED ISSUE #947: [prometheus-pushgateway] Setting priorityclassname
**Is your feature request related to a problem? Please describe.**
We'd like to set PriorityClassName for the pushgateway deployment.

**Describe the solution you'd like**
#

### PR DESCRIPTION
**Pull request by blame19**
Signed-off-by: Matteo Maponi <matteo.maponi@gmail.com>

#### What this PR does / why we need it:
This adds the priorityClassName argument for prometheus-pushgateway.

#### Which issue this PR fixes
  - fixes #947 

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-pushgateway/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.3.0"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 1.9.0
+version: 1.9.1
```
Comment (monotek) [inline charts/prometheus-pushgateway/Chart.yaml:5]: new feature should raise minor version

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — APPROVED** [review]

**Comment by blame19** [comment]
@gianrubio Could you review?

**Comment by monotek** [comment]
@prometheus-community/helm-charts-admins 
please review :)

**Comment by blame19** [comment]
@gianrubio @prombot  Could you review?

**Review by zanhsieh — APPROVED** [review]

**Comment by zanhsieh** [comment]
@cstaud 
Could you review and merge please?

**Comment by monotek** [comment]
@torstenwalter @scottrigby 
please have a look.

**Comment by blame19** [comment]
@torstenwalter @scottrigby @gianrubio 
please have a look.

**Comment by blame19** [comment]
@torstenwalter @scottrigby @gianrubio
please have a look.

**Comment by blame19** [comment]
@torstenwalter @scottrigby @gianrubio @zanhsieh @monotek @cstaud 
What can I do to get this looked on?

**Review by torstenwalter — APPROVED** [review]

**Comment by torstenwalter** [comment]
Reviewed and merged as admin in accordance to https://github.com/prometheus-community/helm-charts/blob/main/PROCESSES.md#review-process

Sorry that it took so long.

**Pull request merged**